build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml

```diff
@@ -1146,12 +1130,27 @@
       <summary>Gets the index of the item the element was prepared for.</summary>
       <returns>The index of the item the element was prepared for.</returns>
     </member>
-    <member name="T:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost" />
+    <member name="T:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost">
+      <summary>A helper to coordinate interaction between the <see cref="Microsoft.UI.Xaml.Controls.ItemsRepeater?text=ItemsRepeater" /> and ScrollViewer.
+Use the ItemsRepeaterScrollHost if your app will run on versions of Windows prior Windows 10 1809 (Build 17763).  If your app will only run on versions of Windows 1809 or higher, there is no need to use this control.</summary>
+    </member>
     <member name="M:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.#ctor" />
-    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.CurrentAnchor" />
-    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.HorizontalAnchorRatio" />
-    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.ScrollViewer" />
-    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.VerticalAnchorRatio" />
+    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.CurrentAnchor">
+      <summary>The currently chosen anchor element to use for scroll anchoring.</summary>
+      <returns>The most recently chosen UIElement for scroll anchoring after a layout pass, or null.</returns>
+    </member>
+    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.HorizontalAnchorRatio">
+      <summary>Determines the horizontal position of the ScrollViewer's anchor point with respect to the viewport. By default, the ScrollViewer selects an element as its <see cref="Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.CurrentAnchor?text=CurrentAnchor" /> by identifying the element in its viewport nearest to the anchor point.</summary>
+      <returns>A normalized value (0.0 to 1.0). The default is 0.0.</returns>
+    </member>
+    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.ScrollViewer">
+      <summary>Gets or sets the ScrollViewer to host.</summary>
+      <returns>A ScrollViewer that contains one or more <see cref="Microsoft.UI.Xaml.Controls.ItemsRepeater?text=ItemsRepeater" /> controls.</returns>
+    </member>
+    <member name="P:Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.VerticalAnchorRatio">
+      <summary>Determines the vertical position of the ScrollViewer's anchor point with respect to the viewport. By default, the ScrollViewer selects an element as its CurrentAnchor by identifying the element in its viewport nearest to the anchor point.</summary>
```

Comment:
`<see cref="Microsoft.UI.Xaml.Controls.ItemsRepeaterScrollHost.CurrentAnchor?text=CurrentAnchor" />` is used instead of CurrentAnchor for the horizontal direction above.